packages/expo/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "expo-datadog",
+  "version": "45.0.0",
```

Comment:
so version will match Expo version? We are going to match only the major version, but minor and patch versions are completely on our own and don't depend on Expo?

Reply:
Yes. 
I believe that since expo is going to break a lot of things related to config plugins in the next versions we need to help users know which versions they can use, and we'll want to keep adding features for previous Expo SDK versions.
I think this is the easiest way of doing it for us.